ajax/libs/PageCSS/package.json

```diff
@@ -0,0 +1,28 @@
+
+  {
+  "name": "pagecss",
+  "version": "0.3.0",
+  "author": "Andrew Chilton",
+  "description": "Beautiful CSS for single page sites - simple all the way up!",
+  "main": "dist/page.css",
+  "style": "dist/page.min.css",
+  "repository": "chilts/pagecss",
+  "dependencies": {
+    "node-sass": "^4.5.3"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/chilts/pagecss.git",
+    "fileMap": [
+      {
+        "basePath": "/dist",
+        "files": [
+          "page.css",
+          "page.css.map",
+          "page.min.css",
+          "page.min.css.map"
```

Comment:
We should use a single pattern `"**/*"` to grab all the files instead of list them manually as we need them all.